File: luci-app-honk/luasrc/model/cbi/honk/route.lua
local honk = require "luci.model.honk_tools"
local m, s

m = Map("honk", translate("Routing Settings"), translate("Configure routing rules for HONK."))

local route_file = "/etc/honk/config.d/route.dae"

s = honk.init_editor(m, "route")
honk.add_editor(s, route_file, "routeconf", translate("Route Configuration"))

return m

File: luci-app-honk/root/www/luci-static/resources/view/honk/route.js
'use strict';
'require honk.common as honk';

return honk.createConfigFileView(
	'/etc/honk/config.d/route.dae',
	_('Routing Settings'),
	_('Configure routing rules for HONK.'),
	_('Route Configuration'),
	_('Routing configuration saved and service reloaded.')
);
